plugin.yaml

```diff
@@ -40,6 +40,16 @@ node_types:
                 A dictionary of parameters allowed by docker-py to the
                 create_container function.
               default: {}
+            daemon_client:
```

Comment:
Once it'll be in master i will cut propose the same feature to 3.2 branch. Does it make sense for you?